src/Native/System.Security.Cryptography.Native/pal_cms.h

```diff
@@ -61,3 +61,49 @@ extern "C" int CryptoNative_CmsGetRecipientStackFieldCount(CmsRecipientStack* re
 Shims the CMS_RecipientInfo_free method
 */
 extern "C" void CryptoNative_CmsRecipientDestroy(CMS_RecipientInfo* recipient);
+
+/*
+Initializes a CMS_ContentInfo with the cipher given by the oid in algorithmOid and returns a pointer
+to it. 
```

Comment:
Since calling this method has turned this CMS message into an Enveloped message, maybe it should say that somewhere in the description.  In fact, the method name should maybe be Envelope instead of Encrypt, since there is an encrypted-data type that is different than the enveloped-data type.

CryptoNative_CmsInitializeEnvelope?